### PR TITLE
fix(filter-fields): remove overwritten font size from filter fields css

### PR DESCRIPTION
**Pull request by SMahil**
remove overwritten font size from filter fields css so that base font size would work

### CODE REVIEW

File: src/app/filters/filter-fields.component.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
     background-color: #fff; //@color-pf-white;
     background-image: none;
     color: #8b8d8f; //@color-pf-black-500;
-    font-size: 12px;
+    //font-size: 12px;
```
Comment (dlabrecq) [inline src/app/filters/filter-fields.component.scss]: This looks like an older file. New code has been added for the type-ahead feature, which also adds another font-size property.

If you want to remove the font-size, you'll need to do it in two places -- in the latest file, that is.

  .btn-default {
    font-size: 12px; <<<<<<<<<<<<<<<<<<
  }
  .typeahead-input-container {
    position: relative;
    padding-right: 0px;
    .caret {
      position: absolute;
      z-index: 2;
      top: 10px;
      right: 5px;
    }
  }

Comment (SMahil) [inline src/app/filters/filter-fields.component.scss]: @dlabrecq changes are done. Please review.

**Comment by joshuawilson** [comment]
@dlabrecq Please review

**Comment by joshuawilson** [comment]
@SMahil please rebase

**Comment by dlabrecq** [comment]
Fabric-ui now uses the filter-fields component from patternfly-ng. This component no longer exists in ngx-widgets. Therefore, I've applied the fix in that repo, instead.

See https://github.com/patternfly/patternfly-ng/pull/89

**Pull request closed without merging**